lexos/processors/prepare/scrubber.py

```diff
@@ -13,6 +13,48 @@
     general_functions as general_functions
 
 
+def get_special_char_dict_from_file(mode: str) -> Dict[str, str]:
+    """Builds special character conversion dictionaries from resource files.
+
+    :param mode: A string which specifies which character set to use.
+    :return: A dictionary with all of the character entities in the chosen
+        mode mapped to their unicode versions.
+    """
+
+    conversion_dict = {}
+
+    if mode == "MUFI-3":
+        filename = constants.MUFI_3_FILENAME
+    elif mode == "MUFI-4":
+        filename = constants.MUFI_4_FILENAME
+    else:
+        return conversion_dict
```

Comment:
https://zen-of-python.info/errors-should-never-pass-silently.html#10

Errors should never pass silently.